pyproject.toml

```diff
@@ -1,30 +1,32 @@
-[tool.poetry]
+[project]
 name = "dac7"
 version = "2024.4.0"
 description = "Tooling for DAC7"
 readme = "README.md"
 license = "LICENSE.md"
 authors = [
-    "Nicolas Bonnotte <nicolas.bonnotte@backmarket.com>",
+  {name="Nicolas Bonnotte", email="nicolas.bonnotte@backmarket.com>"},
 ]
-packages = [
-  { include = "dac7", from = "cli/src" },
-]
-include = [
-    "cli/src/dac7/schemas/*.xsd"
+requires-python = ">=3.11,<3.14"
```

Comment:
It is not recommended to set an upper bound for Python version, as this will prevent using the library once 3.14 is out, even if it is in fact compatible.